src/common/dom/load_resource.ts

```diff
@@ -22,6 +22,8 @@ const _load = (
         (element as HTMLScriptElement).async = true;
         if (type) {
           (element as HTMLScriptElement).type = type;
+          // https://github.com/home-assistant/frontend/pull/6328
+          (element as HTMLScriptElement).crossOrigin = "use-credentials";
```

Comment:
This should have been `anonymous` instead of `use-credentials`, we will now also send credentials to external resources instead of only to the same origin.

Besides, if you send credentials to an external resource, the 'Access-Control-Allow-Origin' header in the response can not be a wildcard '*'. This will break most if not all CDN's.

Reply:
Oops.

Reply:
Hmmm. In my testing locally, `anonymous` did not cause the credentials to be sent with Safari for the same domain and `use-credentials` did. I did not actually test with cross origins (CDNs).

Reply:
Maybe we should test if it is a relative url, and in that case add `use-credentials`?

Reply:
Sounds good to me. Do we have unit tests for this function? Because it seems worth testing this behavior if we're making it controlled by the url path.

Reply:
We don't.